### ISSUE TITLE
README.md file fails to render

### ISSUE THREAD

**Issue by GR360RY** [ansible/galaxy-issues#84]
Role was imported successfully, but the `README` tab is empty. Reimporting role or removing/adding to galaxy doesn't help. 

To reproduce: create an empty README.md file with {{ foo }} string.

**Comment by gregdek** [comment]
Some clarity here would help. :)

Do you mean that an empty README.md file causes the issue? Or do you mean that any field with mustaches around it causes the issue?

**Comment by GR360RY** [comment]
The latter is the problem: any field with moustaches brakes the README. Removing the line with moustaches and reimporting the role fixes the issue. You can reproduce it easily - jura take a perfectly working role and add `{{ foo }}` string in any place.

**Comment by gregdek** [comment]
Thanks for clarifying. We'll take a look at getting this into the upcoming bugfix release.

**Comment by chouseknecht** [comment]
This is working for me. It may have been fixed as part of the unicode character issue.

Attached are screen shots showing my test case. I added the string {{ foo }} to the bottom of README.md in ansible-role-sendmail. Then I reimported the role into Galaxy, and the README.md imports renders just fine.

Let me know, if I'm not understanding the issue.

Here's the raw README.md:

![raw_readme](https://cloud.githubusercontent.com/assets/1282865/10732001/682531d8-7bce-11e5-9b61-b2c696738d0b.png)

Here's README.md rendered in Galaxy:

![rendered_readme](https://cloud.githubusercontent.com/assets/1282865/10732004/6dac3e26-7bce-11e5-9fa5-97da84a2a8c0.png)
